gits/test/capabilities_test.py

```diff
@@ -482,16 +482,18 @@ def test_cap_kcub1(self):
         """
         term = self._terminal
 
+        # The x position of the cursor is at the left-most corner.
```

Comment:
Let's remove this and the following three comments. The code is very straightforward.